v1/dnscrypt-resolvers.csv

```diff
@@ -91,7 +91,10 @@ dnscry.pt-adelaide-ipv4,dnscry.pt-adelaide-ipv4,"DNSCry.pt Adelaide - DNSCrypt,
 dnscry.pt-adelaide-ipv6,dnscry.pt-adelaide-ipv6,"DNSCry.pt Adelaide - DNSCrypt, no filter, no logs, DNSSEC support (IPv6 server)",,,,2,yes,yes,no,[2602:fba1:a00::100:1],2.dnscrypt-cert.dnscry.pt,56B7:CC14:0DB3:98E3:1084:87B2:4A58:ED64:8D2B:8B0B:8FFA:AB6C:588B:939B:5204:DD6E,
 dnscry.pt-allentown-ipv4,dnscry.pt-allentown-ipv4,"DNSCry.pt Allentown - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,23.137.253.24,2.dnscrypt-cert.dnscry.pt,DD9D:1823:BB9D:5C88:CA59:C3C2:E467:539B:8524:E83D:71D8:3BB2:62E8:F639:9CF6:70A4,
 dnscry.pt-allentown-ipv6,dnscry.pt-allentown-ipv6,"DNSCry.pt Allentown - DNSCrypt, no filter, no logs, DNSSEC support (IPv6 server)",,,,2,yes,yes,no,[2602:fc24:19:74b0:5285::12],2.dnscrypt-cert.dnscry.pt,DD9D:1823:BB9D:5C88:CA59:C3C2:E467:539B:8524:E83D:71D8:3BB2:62E8:F639:9CF6:70A4,
-dnscry.pt-amsterdam-ipv4,dnscry.pt-amsterdam-ipv4,"DNSCry.pt Amsterdam - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,45.86.162.110,2.dnscrypt-cert.dnscry.pt,6E5C:573C:9A33:687D:DDD1:3F61:FFAF:4EA7:59E7:6106:1B5B:8C88:59DF:32A4:E391:9CAF,
+dnscry.pt-amsterdam-ipv4,dnscry.pt-amsterdam-ipv4,"DNSCry.pt Amsterdam - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,198.140.141.46,2.dnscrypt-cert.dnscry.pt,5A9B:69F3:B181:7B8A:C0E8:18C5:0E97:52A1:D690:C52D:FF92:C8DA:70E4:7551:54E6:19BB,
+dnscry.pt-amsterdam-ipv6,dnscry.pt-amsterdam-ipv6,"DNSCry.pt Amsterdam - DNSCrypt, no filter, no logs, DNSSEC support (IPv6 server)",,,,2,yes,yes,no,[2a03:94e3:222b::1032],2.dnscrypt-cert.dnscry.pt,5A9B:69F3:B181:7B8A:C0E8:18C5:0E97:52A1:D690:C52D:FF92:C8DA:70E4:7551:54E6:19BB,
+dnscry.pt-amsterdam02-ipv4,dnscry.pt-amsterdam02-ipv4,"DNSCry.pt Amsterdam 02 - DNSCrypt, no filter, no logs, DNSSEC support (IPv4 server)",,,,2,yes,yes,no,45.86.162.110,2.dnscrypt-cert.dnscry.pt,6E5C:573C:9A33:687D:DDD1:3F61:FFAF:4EA7:59E7:6106:1B5B:8C88:59DF:32A4:E391:9CAF,
+dnscry.pt-amsterdam02-ipv6,dnscry.pt-amsterdam02-ipv6,"DNSCry.pt Amsterdam 02 - DNSCrypt, no filter, no logs, DNSSEC support (IPv6 server)",,,,2,yes,yes,no,[2a07:efc0:1001:a5ce::b4b4],2.dnscrypt-cert.dnscry.pt,6E5C:573C:9A33:687D:DDD1:3F61:FFAF:4EA7:59E7:6106:1B5B:8C88:59DF:32A4:E391:9CAF,
```

Comment:
Key `6E5C:573C:9A33:687D:DDD1:3F61:FFAF:4EA7:59E7:6106:1B5B:8C88:59DF:32A4:E391:9CAF` is renamed from `dnscry.pt-amsterdam-ipv4` to `dnscry.pt-amsterdam02-ipv4`.